src/Tizen.Wearable.CircularUI.Forms.Renderer/Shell/ShellRenderer.cs

```diff
@@ -72,8 +76,8 @@ void UpdateFlyoutMenu()
             var flyoutItems = (Element as IShellController).GenerateFlyoutGrouping();
             if (flyoutItems.Count > 0)
             {
-                InitializeNavigationDrawer();
```

Comment:
It need because _navigationView is disposed when flyoutItems.Count was 0 by `DeinitializeNavigationDrawer`

Reply:
Fixed to be called when `flyoutItems.Count` is bigger than 0.